core/common/assets/js/api/modules/command-data.js

```diff
@@ -202,16 +202,19 @@ export default class CommandData extends CommandBase {
 	}
 
 	/**
-	 * Called after apply() failed.
-	 *
-	 * @param e
+	 * @param {BaseError} e
 	 */
+	applyAfterCatch( e ) {
+		e.notify();
+	}
+
 	onCatchApply( e ) {
 		// TODO: If the errors that returns from the server is consistent remove the '?' from 'e'
-		const status = e?.data?.status || 0;
+		const httpErrorCode = e?.data?.status || 501;
```

Comment:
Why 501 and not 500?

Reply:
The default exception converted to error 500.

In case you add more generic error(s) (by extending `Elementor\Data\V2\Base\Exceptions\Data_Exception`)
and you forget to give the right HTTP code, the error will be '501 not implemented'.